source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,237 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Aggregation operations process data in your MongoDB collections and return computed results. The **aggregation pipeline**
+is like a factory assembly line. Along the line are assembly stations, called **aggregation stages** . Like an assembly
+station configured with specific tooling to perform its task, each ``stage`` uses **expressions** to perform some
+operation or combination of operations.
```

Comment:
The sentences switch between extended metaphor and simile which makes it a little more difficult to read. I would stick to one of the two throughout the paragraph. E.g.

Extended metaphor:
The aggregation pipeline is a factory assembly line.  The sequentially-arranged assembly stations are aggregation stages. Each assembly station at which a specific task is performed is an aggregation stage with a specific operation or combination of operations. 

Simile (compare two things):
Aggregation pipelines are like a factory assembly line. Aggregation stages are like the assembly stations that sequentially put together the finished product. Similar to assembly stations, aggregation stages use specific tools called expressions to perform a specific operation or combination of operations.